### DISCUSSION TITLE
Found interesting skills buried in the Python SDK but not documented in the API docs

### DISCUSSION THREAD

**Discussion by EniasCailliau** [deepgram/community#70] [General help] [answered]
Hi, all. 

I'm using DeepGram's Python SDK 2.1.0 to build an integration between DeepGram and Steamship. 

I found exciting language AI skills documented in the Python SDK under `deepgram._types.PrerecordedOptions` and `deepgram._types.TranscriptionOptions`:

```
analyze_sentiment: bool,
sent_thresh: float,
detect_entities:bool, 
```

However, these are not documented in the API-reference. Can I use these skills, are they officially supported? 

Thanks, love the product, keep it up!

**Comment by geekchick** [comment, marked as answer]
Hi @EniasCailliau ! Thank you for reaching out! ..and great observation as well.

Those features are currently in Beta at the moment but will be available for the general public very soon. Once they're available we'll update the API reference docs. 

Once again, thank you for reaching out. If you need anything please feel free to reach out anytime!
